### DISCUSSION TITLE
The members of Venus Master Program Phase 2

### DISCUSSION THREAD

**Discussion by Joss-Hua** [filecoin-project/community#516] [News and Events]
# Venus Master - VenusHub platform sub-program

We are glad to introduce the **Venus Master** program to you as part of our [VenusHub](https://venushub.io). We have received positive feedback from the community. In case you haven't heard, check out the program [**here**](https://venushub.io/master/p2.html).

Venus Master program is a long-term community program and has made much progress as planned.

## This is important

Filecoin's mission is to create a decentralized, efficient, and robust foundation for humanity’s information. Based on this vision, different implementations of Filecoin is making progress.

[Venus](https://github.com/filecoin-project/venus), one of 4 implementations of Filecoin, is moving towards its goal through its distributed architecture. A large number of developers and storage providers would like to participate/contribute, but they can't start or progress effectively due to the lack of necessary support.

In the long run, Venus Masters is crucial to the success of Venus. It helps more storage providers smoothly join the Incubation Center or use Venus to run their Filecoin nodes, and helps more developers understand, familiarize and master Venus to jointly achieve this goal.

## How to work and rewards

For an introduction to the Venus Master program and an overview of its latest phase, please [check here](https://venushub.io/master/p2.html).

## Virtual organization

As pointed out in the project introduction, each stage will be limited to the selection of a fixed number of community members, but regardless of whether there is this label or not, Venus community members are developing towards learning together and helping each other.

## How to apply

- Application is opening
- Candidates will be notified via Slack/Email/WeChat
- Stay tuned for future cohort opportunities!

Comments are welcome in this thread.

**Comment by Joss-Hua** [comment]
## :handshake: Thanks

We will share their profiles with the community on this thread. We are excited to have them support Filecoin SPs / Clients / Developers, etc.

**Comment by Joss-Hua** [comment]
## Meet the Venus Masters

Find them on GitHub, Filecoin Slack and etc if you have any questions!

**Comment by Joss-Hua** [comment]
# Dennis Zou

An old developer who has been in the Internet industry for more than ten years.

![TEHTVS1L6-U01U2M1GZL7-g8cdf2c64dfe-512副本](https://user-images.githubusercontent.com/75723122/139783175-59d5685b-da7d-426e-9f42-1153b11fbe61.jpg)

Wrote code;
Managed the server;
Led the technical team;
Developed large Internet applications;
Also managed large-scale clusters;
Like to study all kinds of new technologies.

**Location:** Shanghai, china
**Slack:** Dennis Zou
**Github:** @denniszou

**Comment by Joss-Hua** [comment]
# HanxianZhai

xxx
xxx
xxx

**Location:** xxx
**Slack:** xxx
**Github:** xxx

**Comment by Joss-Hua** [comment]
# Seantw

xxx
xxx
xxx

**Location:** xxx
**Slack:** xxx
**Github:** xxx

**Comment by Imccccc** [comment]
Technical expert in tranditional Internet industry for more than 6 years.
I am a blockchain enthusiasts and willing to make a positive impact on Filecoin/IPFS network.
![image](https://user-images.githubusercontent.com/3400692/166895017-11290a3d-8911-41b3-81e5-e367e7fe0626.png)


Location: Beijing, China
Slack: Tyler C
Github: @Imccccc

**Comment by Joss-Hua** [comment]
# Sela
### Brief intro
A 2-year-old Filecoin Solo SP is also a Web2 developer.

### Self- image
![1](https://user-images.githubusercontent.com/75723122/206104563-cc8e315e-d80a-4cc9-83d9-8ce1062a8d3f.png)

### Hobby
Like to learn some new technical skills from the community silently.

### Contact
**Location:** NanJing, china
**Slack:** [Sela](https://filecoinproject.slack.com/archives/D03QBKRR171)

**Comment by Joss-Hua** [comment]
# ovy
### Brief intro
OG of Mining.
Many years of development experience in Web2, and 3 years of experience in Filecoin SP. 

### Self- image
![hdImg_4ac8b0a0c60ada0c40ae1f641f3dcd361670398574535](https://user-images.githubusercontent.com/75723122/206135531-e73918d0-ce4d-44ac-a0a4-faeaeb6dec24.jpg)

### Hobby
Research technology.
Like all kinds of new things.

### Contact
**Location:** ShenZhen, China
**Slack:** [ovy](https://filecoinproject.slack.com/team/URPFA7HPB)

**Comment by Joss-Hua** [comment]
# mixman

### Brief intro


### Self- image


### Hobby


### Contact
**Location:** 
**Slack:**
